care/emr/reports/context_builder/data_points/symptom.py

```
@@ -0,0 +1,46 @@
+    SingleUserRelatedContextBuilder,
+)
+
+
+class SymptomsReportFilter(filters.FilterSet):
+    clinical_status = filters.CharFilter(lookup_expr="iexact")
+    verification_status = filters.CharFilter(lookup_expr="iexact")
+
+
+class SymptomsContextBuilder(QuerysetContextBuilder):
+    filterset_class = SymptomsReportFilter
+    __filterset_backends__ = [filters.DjangoFilterBackend]
```

Ruff (0.14.6)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)